app/views/users/posts/index.html.erb

```diff
@@ -0,0 +1,8 @@
+<h1><%= @user.name %> Posts</h1>
+
+<% @posts.each do |post| %>
+  <div class="post">
+    <h2><%= post.title %></h2>
+    <p><%= post.text %></p>
+  </div>
+<% end %>
```

Comment:
- [ ] You already have `posts index` and `posts show` views above, please delete the ones inside the `./app/views/users/posts` folder and Kindly don't repeat yourself (DRY).

Reply:
- [ ] @NoelLincoln Kindly remove these view files in this directory as previously requested, its a duplication of already implemented views.